### PR TITLE
Add new mongodb extension

### PR DESCRIPTION
**Pull request by tuyakhov**
Stubs for [new Mongodb extension](http://docs.php.net/manual/en/set.mongodb.php) according to [New PHP and HHVM Drivers](https://www.mongodb.com/blog/post/call-feedback-new-php-and-hhvm-drivers) and [mongodb docs](https://docs.mongodb.org/ecosystem/drivers/php/).
Please, let me know if my PR good enough and if it could be merged in near future. I haven't submitted contributor license agreement yet, but I will sign it after your approval.
Thanks

### CODE REVIEW

**Comment by moderndeveloperllc** [comment]
@tuyakhov Curious - what did you use to create this?

**Comment by moderndeveloperllc** [comment]
Also, I would go through and take out all the language codes in the "@link" docs: http://php.net/manual/ **ru** /class.mongodb-driver-writeconcernerror.php 
http://php.net/manual/ **en** /mongodb-driver-manager.construct.php

**Comment by tuyakhov** [comment]
@moderndeveloperllc I use a custom script to fetch data from php.net. And PhpStorm ofcourse for small manual changes :)
Yep, i am sorry for language codes. I amend current commit to keep history more clean.
Thanks.
P.S Curious - what did you and other guys usually use to create stubs?

**Comment by bjori** [comment]
The mongodb docs are going through major updates these days -- both in terms of ensuring it matching the implementation (been several changes since they were initially created) and adding more content.
So these may need to be re-generated in few days to get all the updated content.
See https://jira.mongodb.org/browse/PHPC-289


I am curious how these are generally created and how often they are updated..
The PHP.net docs team tried to generate several formats, including a .json based one, that was originally hoped to be used by the IDEs; http://doc.php.net/downloads
It does seem raw .php stubs however is the preferred format.. Wonder how much work it is to export it from the manual :]

**Comment by bjori** [comment]
@tuyakhov thanks btw for working on this! :D

**Comment by tuyakhov** [comment]
@bjori Thanks for mentioning about doc updates. This is my first experience with writing phpstorm stubs and i will be glad to know best practices for it. For about 2 weeks I am working on one project where i am using combination php7 + mongodb. To generate this file i created a custom php script which is just parse html. So after it is executed few manual changes is still needed (it takes me about 2 hours). I did not use .json file because there are few things missed (like "Introduction", etc.). I was planning to update it each time something new will come up. But if you guys have an idea how to keep it up-to-date in more efficient way, please let me know.

File: mongodb.php [hunk 1]
```diff
@@ -0,0 +1,1041 @@
...
+ * Documentation taken from http://php.net/manual/en/set.mongodb.php
+ * @author Anton Tuyakhov <atuyakhov@gmail.com>
+ */
+
+/**
+ * Unlike the mongo extension, this extension supports both PHP and HHVM and is developed atop the » libmongoc and » libbson libraries. It provides a minimal API for core driver functionality: commands, queries, writes, connection management, and BSON serialization.
+ * Userland PHP libraries that depend on this extension may provide higher level APIs, such as query builders, individual command helper methods, and GridFS. Application developers should consider using this extension in conjunction with the » MongoDB PHP library, which implements the same higher level APIs found in MongoDB drivers for other languages. This separation of concerns allows the driver to focus on essential features for which an extension implementation is paramount for performance.
+ * @link http://php.net/manual/en/set.mongodb.php
+ */
+namespace MongoDB {
```
Comment (neuro159) [inline mongodb.php]: nested namespace declarations look bit unusual, but acceptable.
http://php.net/manual/en/language.namespaces.definitionmultiple.php
https://github.com/php/php-langspec/blob/PHP-5.6/spec/18-namespaces.md#namespaces

**Comment by neuro159** [comment]
Have you submitted a contributor license agreement? If not, please download the PDF from http://www.jetbrains.org/display/IJOS/Contributor+Agreement and then print it out, sign (mention you github account name there), scan and email to contribute@jetbrains.com

**Comment by tuyakhov** [comment]
Done. I've submitted a contributor license agreement. Thank you for quick reply.

**Comment by aris-b** [comment]
@tuyakhov nice ! :-) :+1:

**Comment by dzotos** [comment]
@tuyakhov Good job man

**Pull request merged**